namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml

```diff
@@ -11,3 +11,23 @@ roleRef:
   kind: ClusterRole
   name: admin
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: nomis-api-access-staging
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller
+  namespace: nomis-api-access-staging
+subjects:
+- kind: ServiceAccount
+  name: tiller
+  namespace: nomis-api-access-staging 
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Can you please change this role to simply `admin`?

Reply:
Yes no prob.   FYO I took that from the guide here https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#tiller-rbac-configuration

Reply:
Thanks for the reference, I'll update.